### ISSUE TITLE
Art tooling for Lonnie to preview props in game

### ISSUE THREAD

**Issue by chase-moskal** [ChaseMoskal/Nanoshooter#12]
We need a generic art asset prop viewer entity that allows us to refresh an art asset in-game.

If we're getting crazy, maybe we could have a system that allows us to refresh all art assets while the game is running. Or maybe just one art asset. Or have it on auto-reload where it reloads every 6 seconds.

Or maybe just have a console command for displaying a prop of the given path.

Or maybe accept a query string parameter to display a prop at the given path.

I like that last option the most, then you could just hit refresh whenever you want to see the prop again. Yeah.

**Comment by chase-moskal** [comment]
Hey, let's make a different subclass of `Game` that is an art viewer.

**Comment by lonnie-ralfs** [comment]
I only understood the title but it sounds cool/useful!

**Comment by chase-moskal** [comment]
Done. You can now browse any `.obj` file for in-game preview by supplying the path as a URL querystring.

I added a readme section about this which explains it in simpler terms – _Art viewer: preview art assets_
